### ISSUE TITLE
Add support for subfolders containing mbtiles tiles

### ISSUE THREAD

**Issue by brendan-ward** [consbio/mbtileserver#27]
There is a limit on the number of unique filenames that can be stored and easily used in a given folder.

For a folder structure like:
```
<tileroot>/a.mbtiles
<tileroot>/b/c.mbtiles
```

would be published as:
```
<domain>/a
<domain>/b/c
```

However, a folder name that is the same as a sibling mbtiles file name would not be allowed, e.g., 
```
<tileroot>/a.mbtiles
<tileroot>/a/b.mbtiles
```

**Comment by brendan-ward** [comment]
In progress in `issue27` branch.

TODO:
* [ ] add directory listing per subdirectory instead of top level
* [ ] push to Heroku
* [ ] update Readme examples

**Comment by brendan-ward** [comment]
Resolved via #32
